apps/meteor/app/livechat/server/api/lib/livechat.ts

```diff
@@ -176,6 +176,8 @@ export async function settings({ businessUnit = '' }: { businessUnit?: string }
 			color: initSettings.Livechat_title_color,
 			offlineTitle: initSettings.Livechat_offline_title,
 			offlineColor: initSettings.Livechat_offline_title_color,
+			position: initSettings.Livechat_widget_position || 'right',
+			background: initSettings.Livechat_background,
```

Comment:
`question` Why settings like `guestBubbleBackgroundColor` are not passed here?

Reply:
@tassoevan Not all features were added as WS settings. Some of them only exist in the widget API. There are plans to add these settings to the Livechat Appearance page in the future, but it's not in the scope for 6.7